tests/test-cases/Group6-VIC-Machine/6-5-Create-Validation.robot

```diff
@@ -29,3 +29,24 @@ Suggest resources - Invalid target path
     ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target=%{TEST_URL}/MUCH/DATACENTER --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD}
     Should Contain  ${output}  Suggested datacenters:
     Should Contain  ${output}  vic-machine-linux failed
+
+Default image datastore
+    Log To Console  \nRunning vic-machine create - default image datastore
+    Set Test Environment Variables  ${true}  default  network  'VM Network'
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+    Set Test VCH Name
+
+    Log To Console  \nInstalling VCH to test server...
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target=%{TEST_URL} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD}
+    Log To Console  ${output}
+    Should Contain  ${output}  Using default datastore
+    Should Contain  ${output}  Installer completed successfully
+    Get Docker Params  ${output}  ${true}
+    Log To Console  Installer completed successfully: ${vch-name}...
+    Sleep  10 seconds
+    ${status}=  Get State Of Github Issue  1109
```

Comment:
this is a timing issue that can cause the test case to fail if we don't sleep. for most of the other integration tests this issue check is in the `Install VIC Appliance To Test Server` keyword